Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -21,7 +18,7 @@ namespace HoloToolkit.UI.Keyboard
     ///       To retrieve the input from the Keyboard, subscribe to the textEntered event. Note that
     ///       tapping 'Close' on the Keyboard will not fire the textEntered event. You must tap 'Enter' to
     ///       get the textEntered event.
-    public class Keyboard : Singleton<Keyboard>
+    public class Keyboard : Singleton<Keyboard>, IDictationHandler
```

Comment:
Since this is a Singleton, does it makes sense to rename this to `VirtualKeyboardManager`?
The name also implies that it would handle regular keyboard input as well, when it doesn't.

Thoughts?

Reply:
Possibly. I personally am not very much in favor of name changes as they brake code

Reply:
Shouldn't be an issue here since we use a prefab, and the GUID will map the updated component.

We could also add another class definition above that uses the old name and mark the old name obsolete in case someone was referencing it before.

Reply:
True that. I am using some of the new public methods I introduced, so renaming will break only one application - mine ;) . But I can live with that